modules/ROOT/pages/using-the-api-console.adoc

```diff
@@ -0,0 +1,34 @@
+= Using the API Console 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The API Console is an interactive component that shows information about REST APIs and their methods, and enables you to test the methods by sending requests. The API Console is located in the API Details page. Endpoints are displayed in alphabetical order.
+
+The API Console displays the API specifications and documentation. An API’s specifications and documentation information are critical to the onboarding process and adoption. Developers need to know how to use and integrate APIs. 
```

Comment:
What about rephrasing this to `The API Console displays the API specifications and documentation information that are critical to the onboarding process and adoption` ?